packages/core/stories/layouts.doc.stories.mdx

```diff
@@ -45,6 +45,14 @@ Defines an equally distributed layout system using columns and rows.
   <Story id="core-layout-gridlayout-griditem--grid-item-wrapper" />
 </Canvas>
 
+## [Split layout](/docs/documentation-core-layout-splitlayout--page)
+
+Defines start and end regions for UI elements within a span, such as a button bar
```

Comment:
Maybe change `span` to `container` as there may be confusion with the `span` element?